src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -27,4 +27,31 @@ public Integer getLength() {
     public String getColor() {
         return color;
     }
+
+    @Override
+    public boolean equals(Object object) {
+        if (object.getClass().equals(Rectangle.class)) {
```

Comment:
bring null check back, but put it before class check

if `object` is null, you will get NullPointerException when executing `object.getClass()`

Reply:
done
We have to check if object is null, but before getClass check, right?